webroot/index.php

```diff
@@ -24,14 +24,14 @@
         return false;
     }
 }
-require dirname(__DIR__) . '/config/bootstrap.php';
+require dirname(__DIR__) . '/vendor/autoload.php';
```

Comment:
Getting this code running with `composer create-project` will be tricky right now. Instead I would:

```
git clone git://github.com/cakephp/app.git appskel
cd appskel
git checkout -t -b 3.3 origin/3.3
composer install
```